src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -277,6 +288,24 @@ public function get_context( ?string $store_namespace = null ): array {
 			? $context[ $store_namespace ]
 			: array();
 	}
+	/**
+	 * Returns an array representation of the current element being processed.
+	 *
+	 * The returned array contains a copy of the element attributes.
+	 *
+	 * @since 6.7.0
```

Comment:
Missing `@return`

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/7460/commits/ee716af83e3eabfe423df007f9b6bfd0234723ce.